src/registrar/assets/sass/_theme/_tables.scss

```diff
@@ -25,6 +25,17 @@
     color: color('primary-darker');
     padding-bottom: units(2px);
   }
+
+  th.action-col-custom-width {
+    width: 24.25%;
+  }
+
+  // Fix margins in mobile view
+  @media screen {
```

Comment:
Looks like this media query is missing a value. In any case, you should use the USWDS mixin (eg: `@include at-media('tablet')`)